scripts/tests/export.sh

```diff
@@ -167,9 +169,9 @@ EOF
 "$earthly" prune --reset
 "$earthly" +multi6
 
-docker run --rm earthly-export-test-6:test cat /data | grep "hello my world"
-docker run --rm earthly-export-test-6:test cat /data | grep "aarch64"
-if docker inspect earthly-export-test-6:test_linux_arm64 >/dev/null 2>&1 ; then
+"$frontend" run --rm earthly-export-test-6:test cat /data | grep "hello my world"
+"$frontend" run --rm earthly-export-test-6:test cat /data | grep "aarch64"
+if "$frontend" inspect earthly-export-test-6:test_linux_arm64 >/dev/null 2>&1 ; then
```

Comment:
The fact that this export test is now carried over too is fantastic!

Reply:
Yep :) 

My plan is for nerdctl to follow the same pattern here. It's really nice since both Podman and Nerdctl ensure they are fully compatible with current Docker CLI commands (to the point where many users literally alias `docker=podman`).